advanced-cache.php

```diff
@@ -362,6 +362,11 @@ function add_debug_html_to_output( $debug_html ) {
 	return;
 }
 
+// Never batcache a request with X-WP-Nonce header.
+if ( ! empty( $_SERVER['HTTP_X_WP_NONCE'] ) ) {
+	return;
```

Comment:
would it make sense to have a `batcache_stats` call here? Something like:

```php
batcache_stats( 'batcache', 'x_wp_nonce_skip' );
```

Reply:
Totally. Updated.